src/script/E2EIdentity/E2EIdentityEnrollment.ts

```diff
@@ -82,13 +78,8 @@ export class E2EIHandler {
    * @param params The params to create the grace period timer
    * @returns The singleton instance of GracePeriodTimer
    */
-  public static getInstance(params?: E2EIHandlerParams) {
-    if (!E2EIHandler.instance) {
-      if (!params) {
-        throw new Error('GracePeriodTimer is not initialized. Please call getInstance with params.');
-      }
-      E2EIHandler.instance = new E2EIHandler(params);
-    }
+  public static getInstance() {
+    E2EIHandler.instance = E2EIHandler.instance ?? new E2EIHandler();
```

Comment:
I needed to simplify the API, in order to be able to get the instance while not configure it. 

So now, to enroll, you need to call `getInstance().initialize()` before calling `enroll`